docs/getting_started/requirements.md

```diff
@@ -96,7 +96,7 @@ Optionally if you intend to edit [PNG, SVG, GIF or WEBP files in the Image Edito
 
 ## Asset manager
 
-- `Node.js` 14+, 16+
+- `Node.js` 14+, 16+ (`Node.js` 14+ has reached its End of Life. We recommend upgrading to the higher version to ensure you receive security updates.)
```

Comment:
```suggestion
- `Node.js` 14+, 16+ (`Node.js` 14+ has reached its End of Life. We strongly recommend using the newer version to ensure you receive security updates.)
```
IMHO. "Strongly" because of the security aspect. "Using" because this affects new installations too. "Newer" is slightly clearer I think.